verification/store.go

```diff
@@ -75,3 +70,33 @@ func LoadX509TrustStore(path string) (*X509TrustStore, error) {
 
 	return &trustStore, nil
 }
+
+func validateCerts(certs []*x509.Certificate, path string) error {
+	// to prevent any trust store misconfigurations, ensure there is at least
+	// one certificate from each file.
+	if len(certs) < 1 {
+		return fmt.Errorf("could not parse a certificate from %q, every file in a trust store must have a PEM or DER certificate in it", path)
+	}
+
+	if len(certs) == 1 {
+		// if there is only one certificate, it must be a self-signed cert or
+		// CA cert.
+		if !isSelfSigned(certs[0]) && !certs[0].IsCA {
```

Comment:
Root cert must be self-signed, so if it is not self signed, it is an invalid certificate chain, right? And if it is self signed root cert, we don't care whether it is CA.

```suggestion
		if !isSelfSigned(certs[0]) {
		    return fmt.Error()
		}
```

Reply:
Already updated the logic as Pritesh called out in https://github.com/notaryproject/notation-go/pull/147#discussion_r978098557